das/local_file_storage_service.go

```diff
@@ -90,3 +90,7 @@ func (s *LocalFileStorageService) ExpirationPolicy(ctx context.Context) Expirati
 func (s *LocalFileStorageService) String() string {
 	return "LocalFileStorageService(" + s.dataDir + ")"
 }
+
+func (s *LocalFileStorageService) HealthCheck(ctx context.Context) error {
+	return nil
+}
```

Comment:
Some kind of deeper healthcheck here would be nice, like checking if a temp file can be written to.